projects/cn0577/Readme.md

```diff
@@ -0,0 +1,9 @@
+# CN0577 HDL Project
+
+Here are some pointers to help you:
+  * [Board Product Page](https://www.analog.com/eval-cn0577)
+  * Parts: [SAR ADC, LVDS interface, 18-bit resolution, Sample rate up to 15MSPS](https://www.analog.com/ltc2387-18)
+  * HDL Doc: https://wiki.analog.com/resources/eval/user-guides/circuits-from-the-lab/cn0577/hdl
+  * Project Doc: https://wiki.analog.com/resources/eval/user-guides/cn0577/hdl
```

Comment:
The project doc should be the same link as HDL, without /hdl

Reply:
Done